package.json

```diff
@@ -58,37 +58,42 @@
     "eslint-plugin-react": "^7.11.1",
     "expect.js": "0.3.1",
     "husky": "^1.1.3",
+    "inject-loader": "^4.0.1",
     "jasmine-core": "^3.2.1",
     "jest": "^23.6.0",
     "karma": "^3.0.0",
-    "karma-browserify": "^5.3.0",
     "karma-chai-sinon": "0.1.5",
     "karma-chrome-launcher": "2.2.0",
-    "karma-jasmine": "^1.1.2",
+    "karma-jasmine": "2.0.1",
```

Comment:
Is there a reason this is an explicit dep vs using ^ like the rest?

Reply:
We're about 50/50 with having the `^` in our package.json. We've run into issues with it not actually locking the version down so I tend to use a specific version number.